lib/configproxy.js

```diff
@@ -510,6 +510,12 @@ class ConfigurableProxy extends EventEmitter {
         return;
       }
 
+      // X-Forwarded-Proto may have multiple values, Tornado uses the last but we want the first
+      if (req.headers['x-forwarded-proto']) {
```

Comment:
This will crash if its not set i think, so usage of .get instead is needed, but perhaps will assume it will have a truthy value if set. So perhaps "mykey" in req.headers is the way to go?

Reply:
Oh this isnt python. Hmmm. Does this become undefined or error?

Reply:
As long as `req.headers` exists this will be `undefined` if `x-forwarded-proto` isn't present